python/pyspark/mllib/linalg/distributed.py

```diff
@@ -500,6 +661,25 @@ def numCols(self):
         """
         return self._java_matrix_wrapper.call("numCols")
 
+    def transpose(self):
+        """
+        Transpose this CoordinateMatrix.
```

Comment:
Ah ok, looking at Matrices.scala (the root class) it indicates it shares the same data type but I forgot to look at the Coordinate matrix underneath. Sorry about that.